### PR TITLE
new pkg: add icu-libs & icu-data-full

### LINKED ISSUE #16333: [Wolfi Package Request]: icu-libs & icu-data-full
### What software would you like us to add to wolfi-os. Ideally include a URL to the project and its source.

icu-libs and icu-data-full is needed for localization support.

### which versions of the software should we include?

latest

### Add some justification for why this specific package and versions are important.

I see this mentioned in another issue https://github.com/dotnet/dotnet-docker/issues/3844.

I want to use the minimal aspnet image, but it seems this is a requirement for the base image to properly support localization. If I'm missing a better way to include this please let me know, since newer to the Wolfi OS project itself.

### Please check all that apply

- [X] This package has an un-restrictive license
- [X] The package/versions proposed are actively maintained upstream
- [ ] I am interested in adding this package to Wolfi OS myself
- [ ] I am willing to help maintain this package

### PR DESCRIPTION
**Pull request by tuananh**
add icu-libs and icu-data-full

Fixes: https://github.com/wolfi-dev/os/issues/16333

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [ ] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Comment by tuananh** [comment]
@ajayk can you take a look at this?

**Comment by sheldonhull** [comment]
Is there a blocking issue with this, or is it just capacity for having it reviewed? Thanks for the help 👍🏻

**Comment by sheldonhull** [comment]
Anything I can do to help move this along, or just a time/capacity issue and keep waiting? Appreciate the PR and support.

**Comment by tuananh** [comment]
> Anything I can do to help move this along, or just a time/capacity issue and keep waiting? Appreciate the PR and support.
> 
> I'm not sure if I can easily use this but add the missing package in a docker layer, but if you have a suggestion on an alternative that would unblock I'd be willing to try that as well.

we maintained an internal wolfi pkgs repository in-house at work, for stuff like this :) while waiting upstream to add it.

File: icu.yaml [hunk 1]
```diff
@@ -32,6 +33,15 @@ pipeline:
       expected-sha256: cb968df3e4d2e87e8b11c49a5d01c787bd13b9545280fc6642f826527618caef
       strip-components: 0
 
+  - uses: fetch
+    with:
+      uri: https://github.com/unicode-org/icu/releases/download/release-${{vars.dash-package-version}}/icu4c-${{vars.underscore-package-version}}-data-bin-l.zip
+      expected-sha256: 2acdb1b982228040963d183b2dd9d321252c613e0f4db213d4bbc10417cde569
```
Comment (jdolitsky) [inline icu.yaml:39]:
```suggestion
      expected-sha256: 593c2c0059108eda0ec0bd37b3d643d906599cfee451a3c5077921d77e485bdb
```

**Review by imjasonh — APPROVED** [review]

**Pull request merged**